README.md

```diff
@@ -766,24 +764,32 @@ you need to make sure that `do_stuff_with_table` can handle the new format.
 
 [[back to top](#databricks-labs-ucx)]
 
-#### `dbfs-read-from-sql-query`
+#### `direct-filesystem-access-in-sql-query`
 
-DBFS access is not allowed in Unity Catalog, so if you have code like this:
+Direct filesystem access is deprecated in Unity Catalog.
+DBFS is no longer supported, so if you have code like this:
 
 ```python
-df = spark.sql("SELECT * FROM parquet.`/mnt/foo/path/to/file`")
+df = spark.sql("SELECT * FROM parquet.`/mnt/foo/path/to/parquet.file`")
 ```
 
 you need to change it to use UC tables.
 
 [[back to top](#databricks-labs-ucx)]
 
-#### `dbfs-usage`
+#### `direct-filesystem-access`
 
-DBFS does not work in Unity Catalog, so if you have code like this:
+Direct filesystem access is deprecated in Unity Catalog.
+DBFS is no longer supported, so if you have code like this:
 
 ```python
-display(spark.read.csv('/mnt/things/e/f/g'))
+display(spark.read.csv('/mnt/things/data.csv'))
```

Comment:
for spark, "reading a file" usually assumes reading a folder with multiple files of the same type, potentially partitioned.

Reply:
partitioned file is just a folder